CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Introduce `BenchmarkRepository`and `StudioBenchmarkRepository`
 - Add `create_project` bool to `StudioClient.__init__()` to enable users to automatically create their Studio projects
 - Add progressbar to the `Runner` to be able to track the `Run`
+- Get all text chunks of a document using the `DocumentIndexClient`.
```

Comment:
We should also mention the method for retrieving the chunks. E.g. like this:
```suggestion
- Add `method DocumentIndexClient.chunks()` for retrieving all text chunks of a document.
```

Reply:
Changed as suggested